applications/luci-app-p910nd/Makefile

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2008-2014 The LuCI Team <luci@lists.subsignal.org>
+# Copyright (C) 2024 OpenWrt.org
```

Comment:
Please assign your own copyright, not OpenWrt.org

Reply:
@systemcrash  Also please assign you or jow~ as a maintainer. It would be nice to have all packages with a specified maintainer.